README.md

```diff
@@ -9,7 +9,7 @@ AccessibilitySnapshots makes it simple to add regression tests for accessibility
 
 ## Getting Started
 
-By default, AccessibilitySnapshot uses [iOSSnapshotTestCase](https://github.com/uber/ios-snapshot-test-case) to record snapshots and perform comparisons. Before setting up accessibility snapshot tests, make sure your project is set up for standard snapshot testing. Accessibility snapshot tests require that the test target has a host application. See the [Extensions](#extensions) section below for a list of other available snapshotting options.
+By default, AccessibilitySnapshot uses [SnapshotTesting](https://github.com/pointfreeco/swift-snapshot-testing) to record snapshots and perform comparisons. The framework also includes support for using [iOSSnapshotTestCase](https://github.com/uber/ios-snapshot-test-case) as the snapshotting engine instead. Before setting up accessibility snapshot tests, make sure your project is set up for standard snapshot testing. Accessibility snapshot tests require that the test target has a host application. See the [Extensions](#extensions) section below for a list of other available snapshotting options.
```

Comment:
Do we think it's worth adding a note in the README under this paragraph something across the lines of:

> ⚠️  Updating to v0.4.0
>
> Until v0.4.0 AccessibilitySnapshot defaulted to using [iOSSnapshotTestCase](/). If you are upgrading from an older version of AccessibilitySnapshot then you will need to update your Podfile to use `pod 'AccessibilitySnapshot/iOSSnapshotTestCase'` instead of `pod 'AccessibilitySnapshot'`.

This equally might work in an MIGRATION.md or something which we could update in the future for breaking changes?

Reply:
I put it in the release notes for 0.4.0. Hopefully that would be one of the first places people would look to see what's new in the release that broke for them.

<img width="1108" alt="Screen Shot 2020-10-16 at 8 55 21 PM" src="https://user-images.githubusercontent.com/1857549/96327813-ea74f300-0ff1-11eb-9605-febf4b372bd6.png">